### DISCUSSION TITLE
Add Chatterino to Windows Package Manager Community Repository (winget)

### DISCUSSION THREAD

**Discussion by criccadamus** [Chatterino/chatterino2#4658] [Ideas] [answered]
Winget is ta package manager that Microsoft is (pushing?). A nice way to update all your packages with one command.
At the moment, Chatterino is not available in this repo, I'd like to be added to it.
Link:
https://learn.microsoft.com/es-es/windows/package-manager/package/repository

**Comment by Felanbird** [comment]
Duplicate of #3020

tldr nobody wants to do it

**Comment by Felanbird** [comment, marked as answer]
@criccadamus someone did it and it was finally added yesterday, `winget install chatterino`
